.ci/build

```diff
@@ -15,51 +15,23 @@
 
 set -e
 
-# For the build step concourse will set the following environment variables:
-# SOURCE_PATH - path to component repository root directory.
-# BINARY_PATH - path to an existing (empty) directory to place build results into.
-if [[ -z "${SOURCE_PATH}" ]]; then
-  export SOURCE_PATH="$(readlink -f $(dirname ${0})/..)"
-else
-  export SOURCE_PATH="$(readlink -f "${SOURCE_PATH}")"
-fi
+source .ci/common.sh
 
+# For the build step concourse will additionally set the following environment variable:
+# BINARY_PATH - path to an existing (empty) directory to place build results into.
 if [[ -z "${BINARY_PATH}" ]]; then
   export BINARY_PATH="${SOURCE_PATH}/bin"
 else
   export BINARY_PATH="$(readlink -f "${BINARY_PATH}")/bin"
 fi
 
-VCS="github.com"
-ORGANIZATION="gardener"
-PROJECT="etcd-druid"
-REPOSITORY=${VCS}/${ORGANIZATION}/${PROJECT}
-
-# The `go <cmd>` commands requires to see the target repository to be part of a
-# Go workspace. Thus, if we are not yet in a Go workspace, let's create one
-# temporarily by using symbolic links.
-if [[ "${SOURCE_PATH}" != *"src/${REPOSITORY}" ]]; then
-  SOURCE_SYMLINK_PATH="${SOURCE_PATH}/tmp/src/${REPOSITORY}"
-  if [[ -d "${SOURCE_PATH}/tmp" ]]; then
-    rm -rf "${SOURCE_PATH}/tmp"
-  fi
-  mkdir -p "${SOURCE_PATH}/tmp/src/${VCS}/${ORGANIZATION}"
-  ln -s "${SOURCE_PATH}" "${SOURCE_SYMLINK_PATH}"
-  cd "${SOURCE_SYMLINK_PATH}"
-
-  export GOPATH="${SOURCE_PATH}/tmp"
-  export GOBIN="${SOURCE_PATH}/tmp/bin"
-  export PATH="${GOBIN}:${PATH}"
-fi
-
 ###############################################################################
 
 VERSION_FILE="$(readlink  -f "${SOURCE_PATH}/VERSION")"
 VERSION="$(cat "${VERSION_FILE}")"
```

Comment:
Is this used anywhere?

Reply:
Looks like this was copied from https://github.com/gardener/etcd-backup-restore/blob/c12013e0773e0adc6264e15ecc752663a7195258/.ci/build#L49-L58 when initially creating the repo, but the env vars were never used in `etcd-druid`'s build process. I think we can safely remove these variables.